src/DataSource/DibiFluentDataSource.php

```diff
@@ -60,7 +60,9 @@ public function getCount(): int
 	 */
 	public function getData(): array
 	{
-		return $this->data ?: $this->dataSource->fetchAll();
+		return $this->data !== []
+			? $this->data
+			: $this->dataSource->fetchAll();
```

Comment:
Could you please keep the ternary operator on one line? Is there a problem in a coding standard point of view?

Reply:
Solved